src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -970,11 +982,13 @@
 				self._nativeChangeOptionBound = nativeChangeOption.bind(null, self);
 				self._focusBound = onFocus.bind(null, self);
 				self._blurBound = onBlur.bind(null, self);
+				self._touchMoveBound = onTouchMove.bind(null, self);
 
 				elSelectWrapper.addEventListener("focus", self._focusBound);
 				elSelectWrapper.addEventListener("blur", self._blurBound);
 				if (!self.options.nativeMenu) {
 					elSelectWrapper.addEventListener("vclick", self._toggleMenuBound);
```

Comment:
Is elSelectWrapper only click event?

Reply:
elSelectWrapper is handling click event when expanding dropdownmenu.
According to my check it was not supporting "touchmove" event that is why I used elOptionContainer